src/features/game/events/landExpansion/cook.ts

```diff
@@ -17,6 +18,22 @@ type Options = {
   createdAt?: number;
 };
 
+type GetReadyAtArgs = {
+  item: ConsumableName;
+  skills: Partial<Record<BumpkinSkillName, number>>;
+  createdAt: number;
+};
+
+export const getReadyAt = ({ item, skills, createdAt }: GetReadyAtArgs) => {
```

Comment:
Do we need to use this inside of `cook()`?

Reply:
😅